src/modules/users/components/TokenActivation/StakesTab/StakesListItem.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+
+import Link from '~core/Link';
+
+import styles from './StakesTab.css';
+
+const MSG = defineMessages({
+  motionUrl: {
+    id: 'users.TokenActivation.TokenActivationContent.StakesListItem.motionUrl',
```

Comment:
Is this the correct id? Shouldn't it be `TokenActivation.StakesTab.StakesListItem`?